azure-pipelines.yml

```diff
@@ -404,18 +226,18 @@ jobs:
   - script: git submodule update --init --recursive
     displayName: 'Clone submodules'
   - script: |
-      rd /Q /S C:\c-dldt-build
-      mkdir C:\c-dldt-build\bin
+      rd /Q /S $(BUILD_DIR)
+      mkdir $(BUILD_DIR)\bin
       rd /Q /S dldt-build
       mkdir dldt-build
-      cd dldt-build
     displayName: 'Create build directory'
   - task: CMake@1
     inputs:
-      workingDirectory: C:\c-dldt-build
+      #cmakeArgs: -G"Visual Studio 15 2017 Win64" -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) -DENABLE_TESTS=ON $(Build.Repository.LocalPath)
```

Comment:
it's better not to keep garbage

Reply:
I agree, but it is under testing now... will be removed later